lib/experimental/fonts-api/bc-layer/class-wp-webfonts.php

```diff
@@ -19,11 +19,25 @@
 /**
  * Class WP_Webfonts
  *
- * BACKPORT NOTE: Do not backport this file to Core.
- * This class exists to give extenders guidance and time to upgrade their code
- * to use the new Web Fonts API.
+ * @deprecated GB 15.1 Use WP_Fonts instead.
  */
-class WP_Webfonts extends WP_Dependencies {
+class WP_Webfonts {
+	/**
+	 * Instance of WP_Fonts.
+	 *
+	 * @var WP_Fonts
+	 */
+	public $wp_fonts;
```

Comment:
Is there a reason why this class property is declared as public?

Reply:
Good catch. Changed in https://github.com/WordPress/gutenberg/pull/50077/commits/9f7b9f9864a7a21d19d8d0aa2a82385261c45168.